tests/gso.test.ts

```diff
@@ -0,0 +1,34 @@
+import dotenv from 'dotenv';
+
+import { expect } from '@jest/globals';
+import { clusterApiUrl } from '@solana/web3.js';
+import { GSO } from '../src';
+
+dotenv.config();
+
+describe('gso-sdk', () => {
+  const client = new GSO(
+    process.env.RPC_URL || clusterApiUrl('devnet'),
+    'confirmed',
+  );
+
+  it('can fetch gsos', async () => {
+    const gsos = await client.getGsos();
+
+    expect(gsos).toBeTruthy();
+    expect(gsos).toBeInstanceOf(Array);
+    expect(gsos.length).toBeGreaterThanOrEqual(1);
```

Comment:
How are you currently testing this? It should be failing on mainnet right now without any gso active